src/core/inputs.jl

```diff
@@ -1,8 +1,8 @@
-
-using ArgParse, OrderedCollections, YAML, MPI
+import ArgParse, YAML, MPI
+import ArgParse: @project_version, @add_arg_table!, parse_args, ArgParseSettings
+import OrderedCollections: LittleDict
```

Comment:
yes, `using` is frown upon like in C++ `using namespace`. It should be fine for scoped things like tests, the REPL or Notebooks.